HinglishULMFiT_AWD_QRNN.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>What does <code>ninja</code> do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Required for qrnn :(